docs/basics.txt

```diff
@@ -635,23 +635,140 @@ bin/gremlin-server.sh ./conf/gremlin-server/http-gremlin-server.yaml
 curl -XPOST -Hcontent-type:application/json -d '{"gremlin":"g.V().count()"}' http://[IP for JanusGraph server host]:8182
 ```
 
+=== JanusGraph Server as Both a WebSocket and HTTP Endpoint
+
+As of JanusGraph 0.2.0, you can configure your gremlin-server.yaml to accept both WebSocket and HTTP connections over the same port. This can be achieved by changing the channelizer in any of the previous examples as follows.
+
+```
+channelizer: org.apache.tinkerpop.gremlin.server.channel.WsAndHttpChannelizer
+```
+
 === Advanced JanusGraph Server Configurations
 
-==== WebSocket versus HTTP
+==== Authentication over HTTP
+There are two ways to authenticate over HTTP with JanusGraph Server: Basic authentication and hash-based messsage authentication code (https://en.wikipedia.org/wiki/Hash-based_message_authentication_code[HMAC]) Auth
 
-JanusGraph Server must be run with a configuration for either WebSocket (for Gremlin Console or other WebSocket programs) or re-configured for HTTP to work with HTTP clients.
+===== HTTP Basic authentication
+To enable Basic authentication in JanusGraph Server include the following configuration in your gremlin-server.yaml.
 
-Currently, there is no way to configure a single running instance of JanusGraph Server to communicate simultaneously with clients talking WebSocket and HTTP.
-However, it is possible to create configuration files (following the steps in this chapter) for both WebSocket and HTTP and start two instances of the JanusGraph Server by pointing it to the various configurations when it is started.
-In this way, the same JanusGraph database instance can be reached through different JanusGraph Servers through both WebSocket and HTTP calls.
-Make sure the port parameter in each yaml configuration is different if starting more than one JanusGraph Server on the same machine:
+```
+ graphs: {
+     credentialGraph: conf/janusgraph-cassandra.properties
```

Comment:
>"graph" instead of "credentialGraph"

This was to specifically show that you can use a separate graph for credentials. I could change it to 

```
graph: conf/janusgraph-cassandra.properties,
credentialGraph: conf/some-other-conf.properties
```
>g.V().count() instead of 2+2/1+1

Not everyone has a `g` variable configured. If either `2+2` or `1+1` are evaluated, you've successfully authenticated.

I'll fix the properties file to hbase properties file.

Reply:
This point in the docs follows the sections where the server with `g` is defined. I think it makes sense for this section to build on that to demonstrate adding authentication and then re-executing the graph query. But I'm more interested in there being a graph query rather than `2+2` than in whether the variable is `g` or something else (to demonstrate the difference) if that makes sense.